src/services/reporting/carSales.ts

```diff
@@ -34,15 +34,12 @@ export const fetchCarSales = async ({
 const envelopeRequest = async (handler): Promise<WithValidationError<null>> => {
   try {
     await handler
-    return {
-      tag: "success",
-      result: null,
-    }
   } catch (error) {
-    return handleValidationError({
-      error,
-      options: { swallowErrors: true },
-    })
+    return handleValidationError(error)
+  }
+  return {
+    tag: "success",
+    result: null,
```

Comment:
Why did you move it outside the `try()`?

Reply:
Because I tried to stick to the way we usually try / catch, but it can remain inside actually 👍